src/hostedPages/activities/form/ConversationalForm.tsx

```diff
@@ -18,6 +19,7 @@ export const ConversationalForm = ({
   questionTypeConfig = {},
   storedAnswers,
   onAnswersChange,
+  autoProgressRadioQuestions = false,
```

Comment:
I usually like verbosity but I think we can just call it `autoProgress` to make it simple and it covers the load. 😄 

```suggestion
  autoProgress = false,
```